src/code/z_demo.c

```diff
@@ -223,7 +223,7 @@ void func_80064824(GlobalContext* globalCtx, CutsceneContext* csCtx, CsCmdBase*
             }
             break;
         case 9:
-            globalCtx->envCtx.unk_EE[3] = 0x10;
+            globalCtx->envCtx.unk_EE[3] = 16;
```

Comment:
I think you should split unk_EE from an array to individual fields, it doesn't look like it makes sense as an array

Reply:
this is a concern for z_kankyo, not object_kankyo. and i still have it as an array over there, so it was probably needed?
we can investigate more, but its not in the scope of this pr